public/stylesheets/style.css

```diff
@@ -67,7 +67,7 @@ body {
   box-shadow: 0 2px 4px rgb(0 0 0 / 10%), 0 8px 16px rgb(0 0 0 / 10%);
   margin: auto;
   padding: 14px 10px 14px 10px;
-  width: 80vw;
+  width: 50vw;
```

Comment:
Changing this applies across every element with the class card - so that's posts on index, users on search, and the input post box -> it might be worth making a separate class for your cards on profile page if this is too wide

Reply:
You might also be able to do something with cards that are inside a certain element, so based off your index.hbs maybe: `.profile-content-container-child > .card` for a more specific reference without having to make a new class

Reply:
I believe we were discussing the idea of having a news feed on one side, in fact may have even done this on a call with you I'm not sure